### PR TITLE
Add change notes to readme

### PR DESCRIPTION
**Pull request by millerjs**
Addresses [DAT-478](https://jira.opensciencedatacloud.org/browse/DAT-478).

Add notes about changes and consideration of breaks in backwards compatibility and downstream effects.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,15 +38,126 @@ characterize the data they represent. Some properties are categorized as `requir
...
+promised to require a data migration.  It is possible that no data
+exists in the GDC that is invalidated by the change, e.g. making a
+field required that has never been left blank.  This should be
+confirmed against the corpus of data and the userbase should be
+notified of a break in backwards-compatibility.
+
+**Breaking Changes include**:
+- Renaming/removing anything that is not a description or comment
+  - Removing an entity schema
+  - Removing a property's allowed `type`
```
Comment (dmiller15) [inline README.md]: What happens if we change a string or number to an enum but capture the values that exist in PROD?

File: README.md [hunk 2]
```diff
@@ -36,15 +38,126 @@ characterize the data they represent. Some properties are categorized as `requir
...
+  - Removing a property's allowed `enum` value
+  - Changing an entity's `category`
+  - Changing an entity's `unique_keys`
+  - Changing an entity's `links`, including `label`, `backref`
+  - Removing a property from an entity schema
+- Changing existence requirements
+  - Adding a property to the `required` list
+  - Changing link `required` from `false` to `true`
+  - Changing link `multiplicity` from `many_to_one` to `one_to_one`
```
Comment (dmiller15) [inline README.md]: I imagine this also applies to changing one_to_many to one_to_one. Is that correct?

Comment (millerjs) [inline README.md]: yup, will add, thanks

File: README.md (hunk 1 above)
Comment (millerjs) [inline README.md]: that should be fine, the old types are still allowed

File: README.md [hunk 3]
```diff
@@ -36,15 +38,126 @@ characterize the data they represent. Some properties are categorized as `requir
...
+`VERSION` file accordingly:
+
+1. MAJOR version when you make incompatible API changes: **Breaking Changes**
+2. MINOR version when you add functionality in a backwards-compatible manner: **Relationship Additions**, **Schema Additions**
+3. PATCH version when you make backwards-compatible bug fixes: **Backwards compatible corrections**
+
 
 ## Contributing
 
 Read how to contribute [here](https://github.com/NCI-GDC/portal-ui/blob/develop/CONTRIBUTING.md).
```
Comment (dmiller15) [inline README.md]: Is this link dead?

Comment (millerjs) [inline README.md]: fixed in https://github.com/NCI-GDC/gdcdictionary/pull/193

**Pull request merged**